### PR TITLE
Removed default quota for subadmins

### LINKED ISSUE #20139: Show quota for subadmins as text and not as dropdown
Subadmins cannot change the system-wide default quota, so the field doesn't work:

subadmin-default-quota

I suggest to simply hide that field when the current user is not an admin (but only a subadmin).

Observed in OC 9.0 pre-alpha / master (f428d31a5d1f91c6e7aa96fd8550c75c02cc0963)

Should hopefully be easy, tagging as junior job

@jancborchardt @MorrisJobke @LukasReschke

### PR DESCRIPTION
**Pull request by prastut**
Fixes #20139

Documenting PR:

Generally I think there are 3 types of users
- Admin [1]
- SubAdmin (which are like admins of some group) [2]
- General Users [3]

Since the "User" interface is only visible for user type [1] & [2], I can just remove the default quota if the user is not of type [1].

### CODE REVIEW

**Comment by prastut** [comment]
@MorrisJobke please review.

**Comment by MorrisJobke** [comment]
I would more change the dropdown to simple text instead of removing it completely for subadmins. Otherwise subadmins only see the "default quota" set for users, but don't know what this value is.

**Comment by MorrisJobke** [comment]
> I would more change the dropdown to simple text instead of removing it completely for subadmins. Otherwise subadmins only see the "default quota" set for users, but don't know what this value is.

cc @owncloud/designers

File: settings/templates/users/part.setquota.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="quota">
 	<!-- Default storage -->
-	<span><?php p($l->t('Default Quota'));?></span>
 	<?php if((bool) $_['isAdmin']): ?>
+		<span><?php p($l->t('Default Quota'));?></span>
```
Comment (prastut) [inline settings/templates/users/part.setquota.php]: @MorrisJobke 
Therefore I moved the `<span><?php p($l->t('Default Quota'));?></span>` inside the check condition of isAdmin. 
Therefore only if the user is admin he/she get's to see "default quota" with the dropdown button.

Comment (MorrisJobke) [inline settings/templates/users/part.setquota.php]: And as Subadmin? This is then completely removed .... I don't really like it - I would prefer just remove the dropdown style and simply provide text with the currently set default value ;)

Comment (prastut) [inline settings/templates/users/part.setquota.php]: @MorrisJobke 
Your suggestion, though valid, would then make the issue invalid.
The discussion is [here]( https://github.com/owncloud/core/issues/20139#issue-114013550).
It purposely says to hide it. Let me know.

Comment (jancborchardt) [inline settings/templates/users/part.setquota.php]: Btw just as a sidenote: We only capitalize the first words, so it should be »Default quota«. :)

**Comment by prastut** [comment]
@MorrisJobke Please review

File: settings/templates/users/part.setquota.php (hunk 1 above)
Comment (prastut) [inline settings/templates/users/part.setquota.php]: @jancborchardt done. :)

File: settings/templates/users/part.setquota.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="quota">
 	<!-- Default storage -->
-	<span><?php p($l->t('Default Quota'));?></span>
+	<span><?php p($l->t('Default quota: '));?></span>
```
Comment (jancborchardt) [inline settings/templates/users/part.setquota.php]: The colon should only be there when the dropdown is not there. It doesn’t need to be in the translation string either. :)

Comment (prastut) [inline settings/templates/users/part.setquota.php]: Umm just to confirm for admin view no colon. For subadmin view colon right?

**Comment by jancborchardt** [comment]
@DeepDiver1975 yo, @owncloud-bot is going nuts with these CLA messages. One per pull request suffices, I already removed a few. ;)

File: settings/templates/users/part.setquota.php (hunk 2 above)
Comment (jancborchardt) [inline settings/templates/users/part.setquota.php]: Exactly. `<?php if((bool) !$_['isAdmin']): ?>:<?php endif; ?>`

Comment (prastut) [inline settings/templates/users/part.setquota.php]: @jancborchardt done.

File: settings/templates/users/part.setquota.php [hunk 3]
```diff
@@ -25,10 +25,6 @@
 		</select>
 	<?php endif; ?>
 	<?php if((bool) !$_['isAdmin']): ?>
-		<select class='quota' disabled="disabled">
-			<option selected="selected">
-				<?php p($_['default_quota']);?>
-			</option>
-		</select>
+		: <?php p($_['default_quota']);?>
```
Comment (MorrisJobke) [inline settings/templates/users/part.setquota.php]: If this equals `none` -> `p($l->t('Unlimited'));` instead of `none`

**Comment by prastut** [comment]
@MorrisJobke Moved to #22929. Please review.

**Pull request closed without merging**